vim-patch:189e24b: runtime(doc): include vietnamese.txt

brianhuster opened the pull request:
Since Vietnamese keymaps in Vim is quite differences from the corresponding input methods, let's document the Vietnamese specifics in vietnames.txt

related: vim/vim#16144

https://github.com/vim/vim/commit/189e24bb1441abe87387a4e21c4387bbf2eac718



vim-patch::9.1.0902

Problem:    
            1. Vim's built-in Vietnamese keymaps have some important
            differences from the correspond Vietnamese input methods, but they
            are undocumented
            2. For some reason, the file vi.po in Neovim is so different
               from that of Vim, and it also contains many mistakes in
               translation, compared to that of Vim

Solution:   
            Add vietnamese.txt
            Copy the file vi.po from Vim to Neovim

https://github.com/vim/vim/commit/8a52587ee05a360cad4d42322200775fc74a4430

brianhuster committed:
vim-patch:189e24b: runtime(doc): include vietnamese.txt

Since Vietnamese keymaps in Vim is quite differences from the
corresponding input methods, let's document the Vietnamese specifics in
vietnames.txt

related: vim/vim#16144

https://github.com/vim/vim/commit/189e24bb1441abe87387a4e21c4387bbf2eac718

brianhuster committed:
vim-patch:336fb22: translation(vi): Update Vietnamese translation

closes: vim/vim#16144

https://github.com/vim/vim/commit/336fb22eaef7977741712d0c4735fc6d65428a4f

justinmk commented:
All these different locale-related help files should have a common prefix like `l10n_`. There are 100s of human languages.

brianhuster commented:
So should I change file names of hebrew.txt, russian.txt and arabic.txt?

justinmk commented:
> So should I change file names of hebrew.txt, russian.txt and arabic.txt?

I'm in favor of that. Any objection @zeertzjq ?

clason commented:
Would need changes to the `vim-patch` script. (Which needs updating anyway, back to the filetype.lua change.)

brianhuster committed:
vim-patch:8a52587: runtime(doc): fix wrong syntax and style of vietnamese.txt

https://github.com/vim/vim/commit/8a52587ee05a360cad4d42322200775fc74a4430

vim-patch:72212c9: runtime(doc): update wrong Vietnamese localization tag

https://github.com/vim/vim/commit/72212c9bea77f14f1e6be703de3c10d70eb2984c

Co-authored-by: Christian Brabandt <cb@256bit.org>

zeertzjq committed:
vim-patch:41afa30: runtime(doc): Add vietnamese.txt to helps main TOC

closes: vim/vim#16177

https://github.com/vim/vim/commit/41afa308d6f420504c47567b494e97a6721afe71

Co-authored-by: h-east <h.east.727@gmail.com>
